### PR TITLE
feat(core): implements the every-at cron syntax

### PR DESCRIPTION
**Pull request by barino86**
This would close out the https://github.com/abichinger/vue-js-cron/issues/44 issue that I created the other day.

### CODE REVIEW

**Comment by abichinger** [comment]
Hello @barino86,

thank you for the PR. It seems like [crontab](https://linux.die.net/man/5/crontab) does not support this syntax so I would prefer if this feature was disabled by default.

Which schedular are you using?

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "babel-jest": "^26.6.3",
     "jest": "^26.6.3",
     "multi-semantic-release": "^2.11.0",
+    "node-gyp": "^10.0.1",
```
Comment (abichinger) [inline package.json:43]: Why is this needed?

**Comment by barino86** [comment]
We are using a kind of home-grown scheduler. We use this library in the UI of our application to help end users build the cron expressions they need, then we use a python library `croniter` to determine the next scheduled date time based on the cron expression.

File: package.json (hunk 1 above)
Comment (barino86) [reply package.json]: removed node-gyp in a new commit.

**Comment by barino86** [comment]
How would you like me to turn it off by default, possibly have a prop that can enable it?

**Comment by abichinger** [comment]
Yeah, a boolean prop would be good.

**Comment by barino86** [comment]
Let me know if there is a different way you would like to see this implemented.

**Comment by barino86** [comment]
Oops, going to fix the test.

**Comment by barino86** [comment]
Fixed.

**Comment by barino86** [comment]
I want to make that prop more dynamic by adding a watch and setting it so that it can be changed on the fly if it needs to be. working on the change for that.

**Comment by barino86** [comment]
Prop is now dynamic and has been added to light and vuetify as well. I think this is ready once you call it good.

**Comment by abichinger** [comment]
Looks good. Thank you for your contribution.

**Comment by barino86** [comment]
Thank you for being open to a new feature on an older version.
